trailblazer/server/schemas.py

```diff
@@ -0,0 +1,7 @@
+from pydantic import BaseModel
+
+
+class AnalysisUpdate(BaseModel):
+    is_visible: bool | None = None
+    comment: str | None = None
+    status: str | None = None
```

Comment:
Maybe this field should be typed with the status enum type. Let me know what you think.

Reply:
Good point 👍